packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -106,6 +115,12 @@ svg.spectrum-Table-sortedIcon {
       text-overflow: ellipsis;
     }
   }
+  &.is-hovered,
+  &.is-cell-focused {
+    .spectrum-Table-menuChevron {
+      display: inline-block;
```

Comment:
Is there supposed to be a menu chevron on hover/focus? I don't see one.

Reply:
O, I see, it's only showing for columns that have both resize AND sort, I'll get that sorted